### DISCUSSION TITLE
No (U)EFI support in bootable images/ISOs

### DISCUSSION THREAD

**Discussion by mradermaxlol** [mikopbx/Core#810] [General ( Обо всём )]
Currently mikopbx ISOs seem to only support legacy BIOS firmware and don't boot in EFI mode at all (as there is no EFI bootloader in the ISO). It would be nice to have (U)EFI support for seamless installation and easier maintenance on modern systems.

Adding support should be pretty straightforward - it's mostly a matter of bundling another bootloader that speaks EFI language.

**Comment by gionag** [comment]
+1 for me
